Cargo.toml

```diff
@@ -33,4 +30,4 @@ tokio = { version = "0.2", features = ["full"] }
 
 [features]
 default = ["vault", "crypto", "random", "snapshot", "primitives", "bincode"]
-sqlite = ["rusqlite"]
+sqlite = []
```

Comment:
I think it's still better to add rusqlite here and add attributes to sqlite module.